client/src/components/PopoverOptionSet/PopoverOptionSet.js

```diff
@@ -16,20 +16,47 @@ class PopoverOptionSet extends Component {
     this.handleSearchValueClear = this.handleSearchValueClear.bind(this);
     this.handleSearchValueChange = this.handleSearchValueChange.bind(this);
     this.handleKeyDown = this.handleKeyDown.bind(this);
+    this.doToggle = this.doToggle.bind(this);
+    this.focusOnTarget = this.focusOnTarget.bind(this);
 
     this.state = {
       searchValue: ''
     };
   }
 
   /**
-   * Pass toggle to parent (props requires a toggle function) and clear the search input
+   * Handle the toggle from the underlying react strap component.
    */
   handleToggle() {
```

Comment:
React strap sends a event object to this method, so we couldn't just reuse it as is..

Reply:
I didn't want to alter the focus when the user clicks out of the popover, because that might cause some other element the user wants to interact with to loose their focus.